init.c

```diff
@@ -433,6 +433,19 @@ int mutt_init(struct ConfigSet *cs, const char *dlevel, const char *dfile,
   mutt_ch_set_charset(charset);
   FREE(&charset);
 
+  char name[256] = { 0 };
+  const char *c_real_name = cs_subset_string(NeoMutt->sub, "real_name");
+  if (!c_real_name)
+  {
+    struct passwd *pw = getpwuid(getuid());
+    if (pw)
+    {
+      c_real_name = mutt_gecos_name(name, sizeof(name), pw);
+    }
+  }
+  cs_str_initial_set(cs, "real_name", c_real_name, NULL);
+  cs_str_reset(cs, "real_name", NULL);
+
```

Comment:
Should we turn this block into a function perhaps?

Reply:
yeah, probably a good idea
(added to my easy-todo list)